itests/org.openhab.core.model.script.tests/src/main/java/org/eclipse/smarthome/model/script/actions/ScriptExecutionTest.java

```diff
@@ -42,7 +41,7 @@
     @BeforeClass
     public static void setUp() throws SchedulerException {
         scheduler = new MockScheduler();
-        System.setProperty(StdSchedulerFactory.PROPERTIES_FILE, "quartz-test.properties");
+        System.setProperty(StdSchedulerFactory.PROPERTIES_FILE, "../../../../../target/classes/quartz-test.properties");
```

Comment:
It is working now but uses the same ugly path workaround like the Config Dispatch tests: https://github.com/openhab/openhab-core/pull/995#discussion_r318601336

Reply:
Let's use this workaround and then we'll remove it when we stop using quartz.